wikipedia-dark.user.css

```diff
@@ -4558,10 +4558,11 @@ regexp("https?:\/\/([\w\-]{2,}\.m\.(wiki(pedia|books|news|quote|source|versity|v
   .client-js body.mw-rcfilters-ui-loading .mw-changeslist {
     opacity: .7;
   }
-  kbd.keyboard-key {
+  kbd.keyboard-key, kbd {
     background: var(--gray-18) !important;
     border-color: var(--gray-5) !important;
     color: var(--gray-9a) !important;
+    -webkit-box-shadow: .1em .2em .2em var(--gray-4) !important;
```

Comment:
Please change this to `box-shadow` so it works in all browsers

Reply:
also make ru changes into ru section. other section may not reach this and vice versa

Reply:
Where I need to move it? Tried section ru.wikipedia.org (line 6307) but it doesn't apply.

Reply:
its because youre modding ru mobile, so create one?

Reply:
Got it.